### PR TITLE
Feature/3240 new state for communication error in exposure check

### PR DESCRIPTION
**Pull request by ArturFriesen**
## Description
This PR adds a new card. This card is show when an error during risk detection happens.

## Link to Jira
[3240](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-3240)

## Screenshot
IMG_0010
IMG_0011

### CODE REVIEW

File: src/xcode/ENA/ENA/Source/Workers/ActiveTracing.swift [hunk 1]
```diff
@@ -38,7 +38,7 @@ struct ActiveTracing: Equatable {
 		// Hours are intentionally rounded down.
 		// We could also simply cast this to `Int` (what we actually do here as well)
 		// but we still call rounded(…) to make it more explicit.
-		Int((interval / TimeInterval.SEC_PER_HOUR).rounded(.down))
+		return 24 * 14 //Int((interval / TimeInterval.SEC_PER_HOUR).rounded(.down))
```
Comment (SwissMountainFrog) [inline src/xcode/ENA/ENA/Source/Workers/ActiveTracing.swift:41]: I guess you intentionally left this comment in, right?

File: src/xcode/ENA/ENA/Source/AppDelegate.swift [hunk 2]
```diff
@@ -121,7 +121,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 	}()
 
 	lazy var riskProvider: RiskProvider = {
-		let exposureDetectionInterval = self.store.hourlyFetchingEnabled ? DateComponents(minute: 45) : DateComponents(hour: 24)
+		let exposureDetectionInterval = DateComponents(second: 5)//self.store.hourlyFetchingEnabled ? DateComponents(minute: 45) : DateComponents(hour: 24)
```
[the remark this answers is not in the document]
Comment (ArturFriesen) [reply src/xcode/ENA/ENA/Source/AppDelegate.swift]: Thanks! Removed debugging code.

File: src/xcode/ENA/ENA/Source/Workers/ActiveTracing.swift (hunk 1 above)
Comment (ArturFriesen) [reply src/xcode/ENA/ENA/Source/Workers/ActiveTracing.swift]: Thanks! Removed debugging code.

**Review by SwissMountainFrog — APPROVED** [review]

File: src/xcode/ENA/ENA/Source/Views/Home Screen/Cells/Risk/RiskFailedCollectionViewCell.swift [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+
+import UIKit
+
+protocol RiskFailedCollectionViewCellDelegate: AnyObject {
+	func activeButtonTapped(cell: RiskFailedCollectionViewCell)
+}
+
+final class RiskFailedCollectionViewCell: HomeCardCollectionViewCell {
+
+	weak var delegate: RiskFailedCollectionViewCellDelegate?
```
Comment (KaiTeuber) [inline src/xcode/ENA/ENA/Source/Views/Home Screen/Cells/Risk/RiskFailedCollectionViewCell.swift:28]: missing MARKs

File: src/xcode/ENA/ENA/Source/Views/Home Screen/Cells/Risk/RiskFailedCollectionViewCell.swift [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+
+	@IBOutlet var titleLabel: ENALabel!
+	@IBOutlet var chevronImageView: UIImageView!
+	@IBOutlet var bodyLabel: ENALabel!
+	@IBOutlet var activeButton: ENAButton!
+
+	@IBOutlet var viewContainer: UIView!
+	@IBOutlet var topContainer: UIView!
+	@IBOutlet var stackView: UIStackView!
+	@IBOutlet var riskViewStackView: UIStackView!
```
Comment (KaiTeuber) [inline src/xcode/ENA/ENA/Source/Views/Home Screen/Cells/Risk/RiskFailedCollectionViewCell.swift:38]: private ?

File: src/xcode/ENA/ENA/Source/Models/Home/HomeRiskCellConfigurator/HomeRiskCellConfigurators/HomeFailedCellConfigurator.swift [hunk 5]
```diff
@@ -0,0 +1,166 @@
...
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import UIKit
+
+final class HomeFailedCellConfigurator: HomeRiskCellConfigurator {
+
+	private var previousRiskLevel: EitherLowOrIncreasedRiskLevel?
```
[the remark this answers is not in the document]
Comment (ArturFriesen) [reply src/xcode/ENA/ENA/Source/Models/Home/HomeRiskCellConfigurator/HomeRiskCellConfigurators/HomeFailedCellConfigurator.swift]: Added marks.

File: src/xcode/ENA/ENA/Source/Models/Home/HomeRiskCellConfigurator/HomeRiskCellConfigurators/HomeFailedCellConfigurator.swift [hunk 6]
```diff
@@ -0,0 +1,166 @@
...
+		case .low?:
+			return AppStrings.Home.riskCardLastActiveItemLowTitle
+		case .increased?:
+			return AppStrings.Home.riskCardLastActiveItemHighTitle
+		default:
+			return AppStrings.Home.riskCardLastActiveItemUnknownTitle
+		}
+	}
+
+	var buttonTitle: String {
```
[the remark this answers is not in the document]
Comment (ArturFriesen) [reply src/xcode/ENA/ENA/Source/Models/Home/HomeRiskCellConfigurator/HomeRiskCellConfigurators/HomeFailedCellConfigurator.swift]: Changed to let

File: src/xcode/ENA/ENA/Source/Views/Home Screen/Cells/Risk/RiskFailedCollectionViewCell.swift (hunk 3 above)
Comment (ArturFriesen) [reply src/xcode/ENA/ENA/Source/Views/Home Screen/Cells/Risk/RiskFailedCollectionViewCell.swift:42]: Added marks.

File: src/xcode/ENA/ENA/Source/Views/Home Screen/Cells/Risk/RiskFailedCollectionViewCell.swift (hunk 4 above)
Comment (ArturFriesen) [reply src/xcode/ENA/ENA/Source/Views/Home Screen/Cells/Risk/RiskFailedCollectionViewCell.swift:52]: Not possible. Its accessed by the configurator.

**Review by KaiTeuber — APPROVED** [review]

**Review by janetback — APPROVED** [review]
UA - > this PR deletes texts that are no longer being used  -> approving.

**Pull request merged**